### PR TITLE
AV-140383 Fixing LoadBalancer Service creation with named ports

### PR DESCRIPTION
**Pull request by aaha97**

### CODE REVIEW

File: internal/lib/utils.go [hunk 1]
```diff
@@ -95,38 +101,53 @@ func GetSvcKeysForNodeCRUD() (svcl4Keys []string, svcl7Keys []string) {
...
+		return pods, targetPort
+	}
+	targetPortFound := false
+	if targetPortName.Type == intstr.Int {
+		targetPortFound = true
+		targetPort = int32(targetPortName.IntValue())
 	}
 	for _, pod := range podList {
+		if !targetPortFound {
+			for _, pc := range pod.Spec.Containers {
```
Comment (monotosh-avi) [inline internal/lib/utils.go:138]: I think matching with container Port may not be necessary. Kubernetes doesn't mandate the Pod to have container port specified.
 @chauhanshubham we got a comment regarding this in Antrea and had to remove that check.

**Review by monotosh-avi — APPROVED** [review]

**Comment by aaha97** [comment]
build ako

**Comment by aaha97** [comment]
build ako

**Comment by anish-avi** [comment]
AKO PR Sanity Check failed

**Comment by aaha97** [comment]
build ako

**Comment by anish-avi** [comment]
AKO PR Sanity Check failed

**Comment by aaha97** [comment]
build ako

**Comment by aaha97** [comment]
build ako

**Comment by anish-avi** [comment]
AKO PR Sanity Check succeeded

**Comment by chauhanshubham** [comment]
build ako

**Review by chauhanshubham — APPROVED** [review]

**Pull request merged**